tests/test_proxy_connect.py

```diff
@@ -31,6 +31,7 @@ def start(self):
         self.proc = Popen(
             [
                 sys.executable,
+                "-u",
```

Comment:
mitmproxy changed the code that prints log lines and we now need to disable stdout buffering to get it via Popen.